### PR TITLE
[#1969] Improvement: In all gradle files group dependancies by their destination

### PR DESCRIPTION
**Pull request by orol116**
…destination #1969

### What changes were proposed in this pull request?

Group dependencies by their destination.

### Why are the changes needed?

Change to group dependencies by their destination.

### Does this PR introduce _any_ user-facing change?

no

### CODE REVIEW

File: integration-test/build.gradle.kts [hunk 1]
```diff
@@ -122,12 +121,14 @@ dependencies {
   testImplementation(libs.okhttp3.loginterceptor)
   testImplementation(libs.mysql.driver)
   testImplementation(libs.postgresql.driver)
-  implementation(libs.commons.cli)
   testImplementation(libs.selenium)
   testImplementation(libs.rauschig)
   testImplementation(libs.minikdc) {
     exclude("org.apache.directory.api", "api-ldap-schema-data")
   }
+  implementation(libs.commons.cli)
```
Comment (justinmclean) [inline integration-test/build.gradle.kts:129]: Can you add a blank line above this

Comment (orol116) [reply integration-test/build.gradle.kts:130]: Sure, I've reflected your request 😊

**Comment by justinmclean** [comment]
Did you try and build before submitting? The build has failed as there are some formatting issues, you can see this in the build output above. you can run `./gradlew :api:spotlessApply` to fix but there might be other issues as well.

**Comment by justinmclean** [comment]
Looks like it was a line-ending issue. We use macOS/unix style line endings, not windows line endings.

**Comment by orol116** [comment]
Thanks for your help!

**Comment by jerryshao** [comment]
@orol116 there's some style issues, please see here (https://github.com/datastrato/gravitino/actions/runs/7778681487/job/21208794040?pr=2055), would you please fix it, thanks.

**Comment by orol116** [comment]
@justinmclean Thanks for your help.

**Review by justinmclean — APPROVED** [review]
LGTM - thanks for you contribution

**Review by jerryshao — APPROVED** [review]

**Pull request merged**